spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/ConfigParameter.scala

```diff
@@ -4,4 +4,7 @@ case class ConfigParameter[T](
   val name: String,
   val section: String,
   val default: T,
-  val description: String)
+  val description: String) extends DataFrameOption {
+
+  override val optionName = name.replaceAll("\\.", "\\_")
```

Comment:
but this method is just for SQL options - for SparkConf options, there is still "name"